doc/sample.rst

```diff
@@ -115,21 +116,20 @@ After programming the sample and all prerequisites to the development kit, test
 Troubleshooting
 ===============
 
-Q: I don't see any output log such as "... Next update check in...". I only see "SoftBank FOTA client started".
-
-A: By default, the library does not use LOG. Check that you have linked the alternative verbose library ``CONFIG_SB_FOTA_LOG=y``
+IF you do not see any output log such as "... Next update check in..." and only see "SoftBank FOTA client started", check that you have linked the alternative verbose library ``CONFIG_SB_FOTA_LOG = y``.
+By default, the library does not use LOG.
 
 Dependencies
 ************
 
 This sample uses the following |NCS| libraries:
 
-* nRF Cloud
-* (AT) Shell
+* `nRF Cloud`_
+* `AT shell`_
 
 It uses the following `sdk-nrfxlib`_ library:
 
-* `nrfxlib_nrf_modem`_
+* `Modem library <nrfxlib_nrf_modem_>`_
 
 It uses the following |addon|.
 
```

Comment:
Cellular samples in sdk-nrf have TF-M listed as a "secure firmware component" at the end of their dependency lists. It should probably be added here as well.